pyproject.toml

```diff
@@ -0,0 +1,105 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "ansys-dpf-core"
+version = "0.6.3.dev0"
+description = "A python wrapper for Ansys DPF"
+readme = "README.md"
+requires-python = ">=3.7,<4"
+license = {file = "LICENSE"}
+authors = [{name = "ANSYS, Inc.", email = "ramdane.lagha@ansys.com"}]
+maintainers = [{name = "PyDPF-core developers", email = "pyansys.maintainers@ansys.com"}]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Science/Research",
+    "Topic :: Scientific/Engineering :: Information Analysis",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+]
+
+dependencies = [
+    "packaging==21.3",
+    "psutil==5.9.2",
+    "tqdm==4.64.1",
+    "numpy==1.23.4",
+    "ansys-dpf-gate==0.2.3",
+    "wheel==0.37.1",
+    "chevron==0.14.0",
+    "black==22.10.0",
+    "codespell==2.2.1",
+    "flake8==5.0.4",
+]
+
+[project.optional-dependencies]
+docs = [
```

Comment:
I was trying to be consistent with what we previously had in dpf-core. @PProfizi, if that is OK with you, I will follow @jorgepiloto's suggestion.

Reply:
Hi @GuillemBarroso, sure we can make this consistent with the pyansys guidelines. I believe this will break some things in the CI though.

Reply:
Good, I will make sure to use ``doc`` and ``tests``.

@PProfizi, I will adapt the CI files accordingly. However, as I mentioned in the PR's description, I will also need to modify the actions in pyansys/pydpf-actions since they are called here. I have just checked, and these actions are also used in PyDPF-Post. Are there any other repos using them? How do you think we can proceed in order to change that? It would be great if we could reuse the pyansys/actions that @jorgepiloto implemented here and add something specific to DPF if required.

Reply:
@GuillemBarroso sure you can edit pydpf-actions.
These are only used in pydpf-core and pydpf-post AFAIK.
To change both, what you can do is create a branch on the pydpf-actions repo, then target in "bump" PRs on both PyDPF-Core and PyDPF-Post to check they work.
Sure if you can find find pyansys/actions which can be directly used that would be nice!

Reply:
I think the best approach is that we will leave it as it is for this PR.

Then, I'll work on the actions and I will use ``doc`` and ``tests`` in both PyDPF-Core and PyDPF-Post. 

I am resolving this conversation.